code/HabitTracker/app/src/main/java/com/example/habittracker/Habit.java

```diff
@@ -58,12 +58,37 @@ public String getTitle() {
     }
 
     /**
-     * Gets the displayable title for the habit
-     * @return title
+     * Returns sharing status
+     * @return isPublic
      */
-    public String getTitleDisplay() {
-        return(titleDisplay);
+    public boolean isPublic() {
+        return isPublic;
     }
+
+    /**
+     * Sets sharing status
+     * @param isPublic {boolean}   true if public, false otherwise
+     */
+    public void setPublic(boolean isPublic) {
```

Comment:
Should we have another function called setPrivate() ?

Reply:
mmm I'll rename it. This doesn't really set it to public. it sets it to whatever you pass. So the name is confusing, probably

Reply:
I renamed it to setShareStatus instead of setPublic